README.rst

```diff
@@ -13,12 +13,16 @@ Tribler
 
 *Towards making Bittorrent anonymous and impossible to shut down.*
 
-We use our own dedicated Tor-like network for anonymous torrent downloading. We implemented and enhanced the *Tor protocol specifications* plus merged them with Bittorrent streaming. More info: https://github.com/Tribler/tribler/wiki
-Tribler includes our own Tor-like onion routing network with hidden services based seeding and end-to-end encryption, detailed specs: https://github.com/Tribler/tribler/wiki/Anonymous-Downloading-and-Streaming-specifications
+We use our own dedicated Tor-like network for anonymous torrent downloading.
+We implemented and enhanced the *Tor protocol specifications*.
+More info: https://github.com/Tribler/tribler/wiki
```

Comment:
This should probably be a hyperlink instead of a raw URL.

Reply:
Changed. But for me, personally, a raw URL looks better :)